AerofoilX/GpSystemServices_X.cpp

```diff
@@ -86,6 +92,10 @@ bool GpSystemServices_X::IsTextInputObstructive() const
 
 bool GpSystemServices_X::IsFullscreenPreferred() const
 {
+	#ifdef __MACOS__
```

Comment:
Use `if (GetOperatingSystem() == GpOperatingSystems::kMacOS)` instead, if you're going to do this, but is there a reason this is returning false?

Reply:
No, it was temp thing I did when I was trying to get things starting up. So removed it.